hugo/content/news/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+Title: List of all news pages
+render_pagelist: true
+---
```

Comment:
Here's how this is currently rendered:
https://preview.aclanthology.org/cschoel_rss_and_blog/news/

We should probably use a different layout for this, at least showing the date of publication, and maybe even a preview of the actual news post itself. I guess Hugo might have some useful default templates for that? It should probably go in `hugo/layouts/news/list.html` (or was it `index.html`?).

Reply:
Yes, I guess there should be something. I can play around with the options a bit tomorrow.

Reply:
I added a `list.html` that includes the `Date` and `Description` fields of the blog posts in 8aa3441. It renders like this:

![image](https://github.com/acl-org/acl-anthology/assets/10725211/5e9a9d8e-73fc-41a8-8fa0-c74e864d938c)

Reply:
Nice! I have some ideas for improvement, but that gives us a starting point with all the needed elements :)